app/models/labware_creators/custom_pooled_tubes/csv_file/row.rb

```diff
@@ -47,13 +47,16 @@ class CustomPooledTubes::CsvFile::Row
 
     def initialize(header, index, row_data)
       @header = header
-      @index = index
-      @source = (row_data[source_column] || '').strip.upcase
-      @destination = (row_data[destination_column] || '').strip.downcase
+      super(index, row_data)
+    end
+
+    def initialize_context_specific_fields
+      @source = (@row_data[source_column] || '').strip.upcase
+      @destination = (@row_data[destination_column] || '').strip.downcase
```

Comment:
Why is the source upcased and destination downcased?

Reply:
Don't know, was like that before.
Switching both to upcase as that seems most sensible